components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -295,8 +296,17 @@ public void variantsSetup(Set<String> enabledFields) throws Exception
             { "segregation", "evidence", "inheritance", "interpretation", "effect", "zygosity", "dbsnp", "transcript",
                 "protein", "cdna" },
             { "sanger", "segregation", "evidence", "inheritance", "interpretation", "effect", "zygosity", "dbsnp",
-                "transcript", "protein", "cdna" }
-            };
+                "transcript", "protein", "cdna" },
+            { "chromosome", "sanger", "segregation", "evidence", "inheritance", "interpretation", "effect",
```

Comment:
All this code has been wrong for some time, and you're just perpetuating the mistake. The intention is to allow a single checkbox in the export dialog to select multiple fields.

Add `variant_coordinates` in `fieldIds`, and `{ "reference_genome", "end_position", "start_position", "chromosome"}` in `headerIds`.